users/vieting/experiments/switchboard/ctc/feat/baseline_args.py

```diff
@@ -260,41 +257,6 @@ def get_returnn_config(
             feature_net["subnetwork"]["preemphasis"] = PreemphasisNetwork(alpha=alpha).get_as_subnetwork(
                 source=source_layer
             )
-        if "codecs" in audio_perturb_args:
```

Comment:
Do you remove this because the assumption is that with smaller mu it should not be applied in recognition at all?

Reply:
Yes, and because mu law is no longer applied to the non perturbed segments by default.